### ISSUE TITLE
Clearer indication of mentions in app

### ISSUE THREAD

**Issue by svbeon** [MCMrARM/revolution-irc#28]
So, neither in the navbar at the top (where you scroll between channels) not in the side menu, there isn't any clear indication of what channels you have been mentioned in, they look the same as any channel that have received a message. It'd be great if there were some clear indicator (like changing the colour of the channel name in those menus) that would indicate that you've been mentioned in that channel

**Comment by haarp** [comment]
Good idea. In addition, there could be an arrow or some other sort of indicator in the channel switcher if there is an unread mention in a channel that's currently not visible.
